mtest/libmscore/split/tst_split.cpp

```diff
@@ -42,6 +42,7 @@ class TestSplit : public QObject, public MTest
       void split06() { split("split06.mscx", "split06-ref.mscx", 6); }
       void split07() { split("split07.mscx", "split07-ref.mscx"); }
       void split08() { split("split08.mscx", "split08-ref.mscx"); }
+      void split295207() { split("split295207.mscx", "split295207-ref.mscx", 5) ; }
```

Comment:
to keep numerical oder better move that test down

Reply:
Done